.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug Report
+about: Report a problem using F´
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+|**Field**|**Value**|
+|:---|:---|
+|**_Report Date_**| [DATE] |
+|**_Reported By_**| [USER] |
```

Comment:
I'm wondering if we can remove Report Date / Reported By fields.  90% of the time, this will be the user who opened the issue and the date the issue was opened. In the rare cases where somebody is opening a ticket on behalf of somebody else they can directly note it in the description.